config/redirects

```diff
@@ -8,6 +8,8 @@ raw: docs/compass/ -> ${base}/current/
 raw: docs/compass/manual/aggregation-pipeline-builder -> ${base}/current/aggregation-pipeline-builder/
 raw: docs/compass/current/query/favorite/ -> ${base}/current/query/queries/
 raw: docs/compass/current/dark-mode/ -> ${base}/current/settings/settings-reference/#settings
+raw: docs/compass/master/dark-mode/ -> ${base}/current/settings/settings-reference/#settings
 raw: docs/compass/current/config-file/config-file-options/ -> ${base}/current/settings/config-file/
+raw: docs/compass/master/config-file/config-file-options/ -> ${base}/current/settings/config-file/
```

Comment:
[out of scope]

These updates look good. I think we may want to run this file by the DOP folks at some point to see how we need to refactor this to avoid needing to list each redirect twice (once for `master` and once for `current`). I'm assuming it's something to do with symlinks, but ideally we'd be able to catch all versions with `[*]`. I'm not sure if there's something preventing us from doing that now.

Reply:
Gotcha! I can bring that up in #ask-docs-platform. I’m personally just not super familiar with the different syntax we use to catch different versions and whatnot, but it’s probably better to have them take a look sooner rather than later 👍